time/slots/slottime.go

```diff
@@ -81,6 +82,25 @@ func ToEpoch(slot primitives.Slot) primitives.Epoch {
 	return primitives.Epoch(slot.DivSlot(params.BeaconConfig().SlotsPerEpoch))
 }
 
+// ToForkVersion translates a slot into it's corresponding version.
+func ToForkVersion(slot primitives.Slot) int {
+	epoch := ToEpoch(slot)
+	switch {
+	case epoch >= params.BeaconConfig().ElectraForkEpoch:
+		return version.Electra
+	case epoch >= params.BeaconConfig().DenebForkEpoch:
+		return version.Deneb
+	case epoch >= params.BeaconConfig().CapellaForkEpoch:
+		return version.Capella
+	case epoch >= params.BeaconConfig().BellatrixForkEpoch:
+		return version.Bellatrix
+	case epoch >= params.BeaconConfig().AltairForkEpoch:
+		return version.Altair
+	default:
+		return version.Phase0
+	}
+}
+
```

Comment:
This is a good candidate for a unit test.

Reply:
Added